web_external/js/views/body/CreateStudyView.js

```diff
@@ -46,7 +46,7 @@ isic.views.CreateStudyView = isic.View.extend({
             getInfoCallback: function (type, obj) {
                 if (type === 'featureset.isic_archive') {
                     return {
-                        text: obj.name,
+                        text: (new isic.models.FeaturesetModel(obj)).name(),
```

Comment:
I'd prefer to assign the new object to a named variable to simplify the syntax, but I'm ok if we don't change it too.